### PR TITLE
Added Managing vulnerabilities topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Content based on https://help.stackrox.com/docs/manage-vulnerabilities/

Preview: https://openshift-docs-git-acs-operating-gnelson.vercel.app/openshift-acs/master/operating/manage-vulnerabilities.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer Ready for review.

**Review by bergerhoffer — COMMENTED** [review]
Lots of little things, but overall nice job!

File: modules/create-policies-to-block-specific-cves.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. Click *CVEs* from the *Vulnerability Management* view header.
+. Select the check boxes (leftmost column) for one or more CVEs and then click *Add selected CVEs to Policy* (*Add* icon).
+Or, move the mouse over a CVE in the list, and select the *Add* icon on the right side.
+
+. For *Policy Name*,
+** To add the CVE to an existing policy, select an existing policy from the drop-down list box.
+** To create a new policy, enter the name for the new policy, and select *Create* policy name.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/create-policies-to-block-specific-cves.adoc:18]: So the button contains the policy name, so 
- if the policy is XYZ123, the button appears as, **Create XYZ123**
-  if the policy is ABCD, the button appears as, **Create ABCD**

File: modules/create-policies-to-block-specific-cves.adoc.bak [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/create-policies-to-block-specific-cves.adoc.bak:1]: Thank you, I noticed that the spellchecker I am using is creating these backup files when I fix spellings. I'll remember to delete these from now on. 👍🏼

File: modules/view-recently-detected-vulnerabilities.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//
+// * operating/manage-vulnerabilities.adoc
+:_module-type: CONCEPT
+[id="view-recently-detected-vulnerabilities_{context}"]
+= Viewing recently detected vulnerabilities
+
+The *Recently Detected Vulnerabilities* widget on the *Vulnerability Management* view shows a list of recently discovered vulnerabilities in your scanned images, based on the scan time and CVSS score.
+It also includes information about the number of images affected by the CVE and its impact (percentage) on your environment.
+
+* When you hover over a CVE in the list, you see an overview of the CVE, which includes scan time, CVSS score, description, impact, and whether it's scored by using CVSS v2 or v3.
```
Comment (bergerhoffer) [inline modules/view-recently-detected-vulnerabilities.adoc:11]: @gaurav-nelson Just one it's -> it is that got missed. I'm going to go ahead and merge this PR though if you can tag that onto another one of your PRs if you don't mind.

**Review by bergerhoffer — COMMENTED** [review]
LGTM

**Pull request merged**